choice.rb

```diff
@@ -1,24 +1,26 @@
 require_relative './books/book_ui'
+require_relative './game/game_ui'
 
 class Choice
   OPTIONS = {
     1 => :list_books,
     2 => :list_music_albums,
-    3 => :list_movies,
-    4 => :list_games,
-    5 => :list_genres,
-    6 => :list_labels,
-    7 => :list_authors,
-    8 => :list_sources,
-    9 => :add_book,
-    10 => :add_music_album,
-    11 => :add_movie,
-    12 => :add_game,
+    3 => :list_games,
+    4 => :list_genres,
+    5 => :list_labels,
+    6 => :list_authors,
+    7 => :add_book,
+    8 => :add_music_album,
+    9 => :add_game,
     0 => :exit
```

Comment:
@ZewdieMc you did a great job minimizing the options and removing unnecessary ones. ⚡